api/DHSC.FingertipsNext.Api/DHSC.FingertipsNext.Api.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net9.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <DockerDefaultTargetOS>Linux</DockerDefaultTargetOS>
+        <RootNamespace>DHSC.FingerTipsNext.Api</RootNamespace>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Asp.Versioning.Mvc" Version="8.1.0" />
+        <PackageReference Include="Asp.Versioning.Mvc.ApiExplorer" Version="8.1.0" />
+        <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="9.0.0-rc.2.24474.3"/>
+        <PackageReference Include="Scalar.AspNetCore" Version="1.2.42" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Content Include="..\.dockerignore">
+        <Link>.dockerignore</Link>
```

Comment:
Caused by a merge issue (long story) - its back now.

Reply:
I think entry in the `.csproj` file is gone now though, so I can't actually see the `.dockerignore` in Rider's Solution view. Or the `Dockerfile` now.

Reply:
The Dockerfile shouldn't be in the project folder, it should really live in the /api since there are many projects involved with its creation - its been moved there, along with the .dockerignore file.  It's not referenced in the Rider solution any more as it can't seem to handle files outside of a project.

Reply:
I've add a reference to a linked file so it can stay where it is on the file system, but you can at least see it in the solution.